ambry-frontend/src/test/java/com.github.ambry.frontend/AmbrySecurityServiceTest.java

```diff
@@ -205,6 +208,10 @@ public void testProcessResponse()
     testExceptionCasesProcessResponse(RestMethod.HEAD, callback, blobInfo);
     testExceptionCasesProcessResponse(RestMethod.GET, callback, blobInfo);
 
+    // test Get of a large blob
+    blobInfo = new BlobInfo(getBlobProperties(100000, false, 10000, "image/gif"), null);
```

Comment:
I don't think you should hardcode. Instead use `frontendConfig.frontendChunkedGet... * 3` or something.